### PR TITLE
[misc] Multiple fixes for v0.9

### PR DESCRIPTION
**Pull request by ailzhang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,16 +32,16 @@
...
      -b 0 -e 400 -s 1 \
-     -o ./path/output \
+     -o ./output \
      --gpu-memory 20 \
      -M 460 \
      --shutter-time 0.0 \
      -r 128
  ```
- - Images are in the `rendered` folder. For example, 100 million MPM particles simulated in 8 hours on a V100 GPU:
+ - Images are in the `outuput/` folder. For example, 100 million MPM particles simulated in 8 hours on a V100 GPU:
```
Comment (strongoier) [inline README.md:44]:
```suggestion
 - Images are in the `output/` folder. For example, 100 million MPM particles simulated in 8 hours on a V100 GPU:
```

**Review by strongoier — COMMENTED** [review]
Why move `engine/render_particles.py` out?

**Comment by ailzhang** [comment]
@strongoier There's an import error if I leave it inside, also `render_particles.py` looks more like a script to be run directly than a module inside a package. ;)

**Review by strongoier — APPROVED** [review]
LGTM

**Review by k-ye — APPROVED** [review]
LGTM!

**Pull request merged**